config/software/datadog-agent-integrations.rb

```diff
@@ -0,0 +1,67 @@
+require './lib/ostools.rb'
+
+name 'datadog-agent-integrations'
+
```

Comment:
I think this only declares a dependency for the package itself (i.e. it needs `datadog-agent`), whereas I think what Olivier wants is a dependency in the Omnibus sense (so that we're sure that `datadog-agent-integrations` is built after `datadog-agent`).